fms_core/tests/test_models.py

```diff
@@ -230,40 +233,42 @@ def test_wrong_container_extracted_sample(self):
     def test_original_sample(self):
         with self.assertRaises(ValidationError):
             try:
-                Sample.objects.create(**create_extracted_sample(biospecimen_type='DNA',
+                Sample.objects.create(**create_extracted_sample(sample_kind=self.sample_kind_DNA,
                                                                 volume_used=Decimal('0.01'),
                                                                 container=self.tube_container,
                                                                 extracted_from=self.invalid_parent_sample,
                                                                 individual=self.valid_individual,
                                                                 name="test_extracted_sample_11"))
             except ValidationError as e:
-                self.assertIn("extracted_from", e.message_dict)
```

Comment:
You are fixing the test by not testing the right thing... The problem to be tested here is that the invalid_parent_sample here should not be created if you create it as DNA. This will cascade to this creation failing because the extracted_from field do not exist ...